FitGoal/FitGoal/SuggestedRoutineCell.swift

```diff
@@ -9,7 +9,9 @@
 import UIKit
 
 protocol RoutineDelegate: AnyObject {
-    func displayExercises(routine: Routine)
+    func displayExercises(exercises: [Exercise])
+    
+    func filterExercises(in routine: Routine) -> [Exercise]
 }
```

Comment:
The purpose of this protocol isn't very clear to me, in particular because the names of the functions don't say much. What you want to express at the cell level is that the plus button was pressed by the user. That's the only thing that the cell should care about. It doesn't need to know about what you are gonna do with the routine itself, it only has to communicate that the routine was selected. A more descriptive definition of the protocol would be:

```swift
protocol SuggestedRoutineCellDelegate: AnyObject {
    func userDidAddRoutine(_ routine: Routine)
}
```